src/python/WMCore/Services/WMArchive/DataMap.py

```diff
@@ -13,18 +13,17 @@
 # convert data format under stpes["cmsRun1"/"logArch1"/"stageOut1"]["output"]
 WMARCHIVE_REMOVE_OUTSIDE_LAYER = ["checksum", "dataset"]
 # convert to list from str
-WMARCHIVE_CONVERT_TO_LIST = ["OutputPFN", "lfn"]
```

Comment:
I don't know if this information could be consumed by anyone, but I'd say `OutputPFN` should remain in the WMA doc, no?